src/Result.ts

```diff
@@ -301,8 +256,9 @@ function tryCatch<A, E = unknown>(
 /** Allows some arbitrary side-effect function to be called
  * using the wrapped Ok value. Useful for trace logging.
  *
- * @param f should not mutate its arguments
- *a
+ * @param f should not mutate its arguments use `map` if you
```

Comment:
:pick: Run-on sentence? At least kinda awkward if not (I'm no english major! =P ) Might just be missing a period?

Reply:
Woops. Totally missed a period there, yep. Good catch!